package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "version": "0.1.2",
   "scripts": {
-    "precodegen": "python3 ./scripts/generate.py",
+    "pycodegen": "python3 ./scripts/generate.py",
```

Comment:
Should this script be removed? If I understand things correctly this was helpful to start the codebase but it wont be used anymore

Reply:
(Same with scripts folder)

Reply:
I agree but will be removing it in a separate PR, I also want to spin off an OpenAPI code generator for Polywrap from this and the Coingecko API gave me the insight to make it possible.